ArduCopter/mode.cpp

```diff
@@ -699,6 +699,24 @@ void Mode::land_run_horizontal_control()
     }
 }
 
+// Do normal landing or precision landing if enabled. pause_descent is true if vehicle should not descend
+void Mode::execute_landing(bool pause_descent)
```

Comment:
I'm not a huge fan of this method's name.  "execute_landing" is synonymous with "run_land_controllers" I think so we at least need to differentiate them somehow..